src/utils/kill-browser-process.js

```diff
@@ -43,17 +40,25 @@ async function findProcessWin (processOptions) {
 }
 
 export default async function (browserId) {
-    const processOptions = { arguments: browserId, psargs: '-ef' };
-    const processList    = OS.win ? await findProcessWin(processOptions) : await findProcess(processOptions);
+    let processId = null;
+
+    if (OS.win) {
```

Comment:
Maybe we can move this condition and `findProcessWin` into the `./process` module?